x64: enable VTune support by default (#3821)
bytecodealliance/wasmtime: docs/examples-profiling-vtune.md

### Turn on VTune support

For JIT profiling with VTune, Wasmtime currently builds with the `vtune` feature
enabled by default. This ensures the compiled binary understands how to inform
the `ittapi` library of JIT events. But it must still be enabled at

> runtime--enable runtime support based on how you use Wasmtime:

Reviewer: hmm, it seems the end of this comment after `--` would need to be removed, as it contains `:` but with no extra indication thereafter?

Author: I'll leave it and we can fix it up later if we think of a more clear way to word this.

Reviewer: Oh, i misread it as a list entry as well, my bad. Maybe can reformulate slightly then:

```suggestion
the `ittapi` library of JIT events. But it must still be enabled at
runtime with one of the following:
```
